src/test/java/io/vertx/core/logging/JULLogDelegateTest.java

```diff
@@ -40,10 +41,24 @@
   public static void initialize() throws IOException {
     // Clear value.
     System.clearProperty("vertx.logger-delegate-factory-class-name");
+    LoggerFactory.initialise();
```

Comment:
Yes it's this issue.

Test orders depends on the system, and on the CI we have one (or more) test touching the logging system before the logger tests. As JUL is handled using a JVM singleton (somehow), we can reset it.